ui/tests/integration/components/clients/page/acme-test.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { render, findAll } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import clientsHandler, { LICENSE_START, STATIC_NOW } from 'vault/mirage/handlers/clients';
+import { addMonths, getUnixTime } from 'date-fns';
+import { CLIENT_COUNT } from 'vault/tests/helpers/clients/client-count-selectors';
+import { formatNumber } from 'core/helpers/format-number';
+import { calculateAverage } from 'vault/utils/chart-helpers';
+import { dateFormat } from 'core/helpers/date-format';
+
+const START_TIME = getUnixTime(LICENSE_START);
+const END_TIME = getUnixTime(STATIC_NOW);
+const { statText, charts, usageStats } = CLIENT_COUNT;
+
+module('Integration | Component | clients | Clients::Page::Acme', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    clientsHandler(this.server);
+    this.store = this.owner.lookup('service:store');
+    const activityQuery = {
+      start_time: { timestamp: START_TIME },
+      end_time: { timestamp: END_TIME },
+    };
+    // set this to 0
+    this.activity = await this.store.queryRecord('clients/activity', activityQuery);
+    this.startTimestamp = START_TIME;
+    this.endTimestamp = END_TIME;
+    this.isSecretsSyncActivated = true;
+
+    this.renderComponent = () =>
+      render(hbs`
+      <Clients::Page::Acme
+        @activity={{this.activity}}
+        @versionHistory={{this.versionHistory}}
+        @startTimestamp={{this.startTimestamp}}
+        @endTimestamp={{this.endTimestamp}}
+        @namespace={{this.countsController.ns}}
```

Comment:
where is countsController coming from? I'm surprised it's not failing if it's unset and attempting to access `.ns` and `.mountPath`

Reply:
`countsController` is injected as `controller` in the acme controller